docs/Software & Tools/ColabFold.md

```diff
@@ -0,0 +1,263 @@
+
+# Using a Jupyter notebook
+
+Start by accessing JupyterLab using the Unity OnDemand interface:
+
+[https://ood.unity.rc.umass.edu/pun/sys/dashboard/batch_connect/sys/jupyterlab/session_contexts/new](https://ood.unity.rc.umass.edu/pun/sys/dashboard/batch_connect/sys/jupyterlab/session_contexts/new)
+
+Click on the JupyterLab interactive app and fill out the following fields:
+
+1. For `Partition` choose one of the following: gpu, gpu-long, uri-gpu, gpu-preempt
+2. `Maximum job duration`: 1:00:00
+3. `Memory (in GB)`: 50
+4. `GPU count`: 1
+5. `Modules`: cudnn/cuda11-8.4.1.50 cuda/11.4.0
+
+The fields `CPU thread count` and `Extra arguments for Slurm` can be left blank.
+
+Inside JupyterLab:
+
+* Open the ColabFold.ipynb notebook
+* Choose Python (colabfold) for the kernel
+* Insert your protein sequence next to query_sequence and execute the code in the cell (press SHIFT+ENTER or press the play button in the toolbar above).
+* Run the code in the remaining cells in order to predict the protein structure with the default parameters (see below) and output plots and a visualization of the 3D structure.
```

Comment:
Instead of "(see below)" it might be useful to be more specific, such as "(see **Notes** section below)"